packages/firestore/exp/src/api/database.ts

```diff
@@ -146,7 +146,8 @@ export class Firestore extends LiteFirestore
     if (this._deferredInitialization !== undefined && !this._terminated) {
       throw new FirestoreError(
         Code.FAILED_PRECONDITION,
-        'Persistence cannot be cleared after this Firestore instance is initialized.'
+        'Persistence cannot be cleared after this Firestore instance is ' +
+          'initialized or after it is terminated.'
```

Comment:
This message is now more incorrect. Persistence *can* be cleared after it is terminated. The error message says that it cannot.

Suggested rewording: Persistence cannot be cleared after this Firestore instance is initialized and before it is terminated.

Reply:
"more incorrect" almost sounds like an improvement :) Thanks for catching.